examples/human_pose_estimation_3d/demo.py

```diff
@@ -1,34 +1,54 @@
-from pipeline import SimpleBaselines
-from paz.applications import HigherHRNetHumanPose2D
+from paz.pipelines.keypoints import SimpleBaselines
 from paz.backend.image import load_image
-from linear_model import Simple_Baseline
+from paz.models.keypoint.simplebaselines import SimpleBaseline
```

Comment:
* Ideally we would have from` paz.models import SimpleBaselines`. You can modify the `paz/models/__init__.py `and add your model there directly.
* We should rename the SimpleBaseline Keras model to SimpleBaselines